src/dbaas/logs/alias/alias.iceberg.service.js

```diff
@@ -0,0 +1,20 @@
+angular.module("ovh-api-services").service("OvhApiDbaasLogsAliasIceberg", function (iceberg) {
+    "use strict";
+
+    // No cache here, because items can be shared at any moment by other users
+
+    var aliasResource = iceberg("/dbaas/logs/:serviceName/output/elasticsearch/alias/:aliasId", {
+        serviceName: "@serviceName"
+    }, {
+        create: { method: "POST" },
+        update: { method: "PUT" },
+        remove: { method: "DELETE" },
+        linkStream: { method: "POST", url: "/dbaas/logs/:serviceName/output/elasticsearch/alias/:aliasId/stream" },
+        unlinkStream: { method: "DELETE", url: "/dbaas/logs/:serviceName/output/elasticsearch/alias/:aliasId/stream/:streamId" },
```

Comment:
Those should probably be in a `stream` submodule

Reply:
Probably, but already the case on v6 https://github.com/ovh-ux/ovh-api-services/blob/master/src/dbaas/logs/alias/alias.v6.service.js